src/test_edgepi/unit_tests/test_calibration/test_access_eeprom.py

```diff
@@ -132,52 +132,52 @@ def test_get_message_of_interest(mocker, msg, eeprom):
-                  4:CalibParam(gain = 1.02623188, offset=-0.0179865),
-                  5:CalibParam(gain = 1.02696733, offset=-0.0169919),
-                  6:CalibParam(gain = 1.0248366, offset=-0.0195936),
-                  7:CalibParam(gain = 1.02198715, offset=-0.0179717)}
+dac_dict_calib = {0:CalibParam(gain = 1.0229951270016944, offset= -0.01787674545454656),
+                  1:CalibParam(gain = 1.0233775195153139, offset= -0.019239763636362414),
+                  2:CalibParam(gain = 1.0238480841375301, offset= -0.014646763636360628),
+                  3:CalibParam(gain = 1.021600135, offset= -0.0190202),
+                  4:CalibParam(gain = 1.022255745463092, offset= -0.0165660727272739),
+                  5:CalibParam(gain = 1.0229838854579036, offset= -0.017047454545456923),
+                  6:CalibParam(gain = 1.0247617896158834, offset= -0.01857707272727247),
+                  7:CalibParam(gain = 1.0208361905168861, offset= -0.016279636363636222)}
 
-adc_dict_calib = {0:CalibParam(gain = 0.998441907, offset=0.035948182),
-                  1:CalibParam(gain = 0.998551392, offset=0.03190953),
-                  2:CalibParam(gain = 1.004123580, offset=-0.039846713),
-                  3:CalibParam(gain = 0.996197528, offset=0.05191299),
-                  4:CalibParam(gain = 1.000529260, offset=0.00375247),
-                  5:CalibParam(gain = 0.995760365, offset=0.05806301),
-                  6:CalibParam(gain = 0.993060022, offset=0.09193925),
-                  7:CalibParam(gain = 0.997010247, offset=0.04080028),
-                  8:CalibParam(gain = 0.998583581, offset=-0.004391864),
-                  9:CalibParam(gain = 0.996344494, offset=0.09228059),
-                  10:CalibParam(gain = 0.995856966, offset=0.05618589),
-                  11:CalibParam(gain = 0.997157199, offset=-0.049672102)}
+adc_dict_calib = {0:CalibParam(gain = 1.0047302081335363, offset= -0.0898128361249185),
+                  1:CalibParam(gain = 1.00005219259183, offset= -0.037039036876729624),
+                  2:CalibParam(gain = 0.9952694074595573, offset= 0.022449294616625794),
+                  3:CalibParam(gain = 0.9930298122067595, offset= 0.052559933743999566),
+                  4:CalibParam(gain = 0.9986894662368055, offset= -0.01639367419587856),
+                  5:CalibParam(gain = 0.9962817511793706, offset= 0.012169769827208299),
+                  6:CalibParam(gain = 0.9977479673435563, offset= -0.006497032357507848),
+                  7:CalibParam(gain = 0.9967431970740426, offset= 0.005837911244602978),
+                  8:CalibParam(gain = 1.0093016097792906, offset= -0.05178411804847496),
+                  9:CalibParam(gain = 0.9977895773887948, offset= -0.030203102327682706),
+                  10:CalibParam(gain = 1.0011882603521973, offset= -0.02699318647809985),
+                  11:CalibParam(gain = 0.9988614798071136, offset= -0.010834493145652147)}
```

Comment:
are these randomly generated inputs? why not use lib random?

Reply:
these are not randomly generated inputs. These values are compared with a dummy serialized protobuf value residing in SDK for testing purpose.